bazel/carbon_rules/defs.bzl

```
@@ -75,7 +75,7 @@ def _carbon_binary_impl(ctx):
                 inputs = depset(direct = srcs_reordered, transitive = dep_hdrs),
                 executable = toolchain_driver,
                 tools = depset(toolchain_data),
-                arguments = ["compile", "--output=" + out.path, "--clang-arg=-stdlib=libc++"] + [s.path for s in srcs_reordered] + extra_flags,
+                arguments = ["compile", "--output=" + out.path, "--clang-arg=-stdlib=libc++"] + [s.path for s in srcs_reordered] + extra_flags + ctx.attr.flags,
```

Break up this long line?
```suggestion
                arguments = (["compile", "--output=" + out.path, "--clang-arg=-stdlib=libc++"] +
                             [s.path for s in srcs_reordered] + extra_flags + ctx.attr.flags),
```